src/Interpreters/HashJoin.cpp

```diff
@@ -224,6 +224,7 @@ HashJoin::HashJoin(std::shared_ptr<TableJoin> table_join_, const Block & right_s
     , right_sample_block(right_sample_block_)
     , log(&Poco::Logger::get("HashJoin"))
 {
+    LOG_DEBUG(log, "HashJoin. Datatype: {}, kind: {}, strictness: {}", data->type, kind, strictness);
```

Comment:
The `data->type` is set below (via `chooseMethod`), so it always `EMPTY` here.
But I wanted to update these logs anyway, so I'll fix it.